### PR TITLE
Check if wmsextent is not null

### PR DESCRIPTION
**Pull request by drillinP**
* Ticket : #2327 
* Adding check to test if wmsextent is not null

### CODE REVIEW

File: lizmap/modules/lizmap/classes/qgisProject.class.php [hunk 1]
```diff
@@ -499,10 +499,12 @@ protected function readWMSInformation($qgsLoad)
             }
             $WMSKeywordList = implode(', ', $values);
 
-            $WMSExtent = $qgsLoad->properties->WMSExtent->value[0];
-            $WMSExtent .= ', '.$qgsLoad->properties->WMSExtent->value[1];
-            $WMSExtent .= ', '.$qgsLoad->properties->WMSExtent->value[2];
-            $WMSExtent .= ', '.$qgsLoad->properties->WMSExtent->value[3];
+            if(!is_null($WMSExtent)){
```
Comment (rldhont) [inline lizmap/modules/lizmap/classes/qgisProject.class.php:502]: It is `$qgsLoad->properties->WMSExtent` to test!

**Review by rldhont — CHANGES_REQUESTED** [review]

**Review by rldhont — APPROVED** [review]
